packages/web-react/src/components/UNSTABLE_Header/index.ts

```diff
@@ -0,0 +1,5 @@
+'use client';
+
+export { default as UNSTABLE_Header } from './UNSTABLE_Header';
+export { default as UNSTABLE_HeaderLogo } from './UNSTABLE_HeaderLogo';
+export * as useUnstableHeaderStyleProps from './useHeaderStyleProps';
```

Comment:
❗ we already have a hook by this name. I think it will be a problem if you try to import it from the spirit-web-react/components folder